net.ventoy.VentoyGUI.yml

```diff
@@ -0,0 +1,206 @@
+              - make install_device-mapper
+              - cd libdm && make install_pkgconfig
+              - chmod 755 /app/lib/libdevmapper.so
+              - chmod 755 /app/sbin/dmsetup
+          - name: libcryptsetup
+            sources:
+              - type: git
+                url: https://gitlab.com/cryptsetup/cryptsetup.git
+                tag: v2.8.0
+            config-opts:
+              - --disable-asciidoc
+              - --disable-ssh-token
+            modules:
+              - name: libpopt
+                sources:
+                  - type: git
+                    url: https://github.com/rpm-software-management/popt.git
+                    tag: popt-1.19-release
+              - name: json-c
+                buildsystem: cmake
+                sources:
+                  - type: git
+                    url: https://github.com/json-c/json-c.git
+                    tag: json-c-0.18-20240915
+          - name: keyutils
+            buildsystem: simple
+            sources:
+              - type: archive
+                url: https://src.fedoraproject.org/repo/pkgs/keyutils/keyutils-1.6.1.tar.bz2/sha512/ea6e20b2594234c7f51581eef2b8fd19c109fa9eacaaef8dfbb4f237bd1d6fdf071ec23b4ff334cb22a46461d09d17cf499987fd1f00e66f27506888876961e1/keyutils-1.6.1.tar.bz2
+                sha256: c8b15722ae51d95b9ad76cc6d49a4c2cc19b0c60f72f61fb9bf43eea7cbd64ce
+            build-commands:
+              - make
+              - make LIBDIR=/app/lib BINDIR=/app/bin SBINDIR=/app/sbin INCLUDEDIR=/app/include MANDIR=/app/share/man SHAREDIR=/app/share/keyutils install
+          - name: bytesize
+            sources:
+              - type: git
+                url: https://github.com/storaged-project/libbytesize.git
+                tag: "2.11"
+          - name: fdisk
+            buildsystem: meson
+            config-opts:
+              - "-Dauto_features=disabled"
+              - "-Dbuild-libfdisk=enabled"
+              - "-Dbuild-fdisks=enabled"
+              - "-Dbuild-libuuid=enabled"
+              - "-Dbuild-libblkid=enabled"
+              - "-Duse-tty-group=false"
+              - "-Dlibutil=enabled"
+            sources:
+              - type: git
+                url: https://github.com/util-linux/util-linux.git
+                tag: v2.41.1
+      - name: atasmart
+        buildsystem: autotools
+        sources:
+          - type: git
+            url: https://github.com/libatasmart/libatasmart.git
+            tag: v0.19
+  - name: ventoy_release
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: https://github.com/ventoy/Ventoy/releases/download/v1.1.05/ventoy-1.1.05-linux.tar.gz
+        sha256: 3379c99890359dcff55aab7f7b3286f87c988d1da2fd616e6a9e305fb0a1de9e
+        strip-components: 1
+    build-commands:
+      - cp -r ventoy-1.1.05/* /app/
+  - name: ventoy_gui
+    buildsystem: simple
+    sources:
+      - type: git
+        url: https://github.com/LorenzoIanotto/VentoyUDisks.git
```

Comment:
is this affiliated to ventoy? Because the location doesn't look like it is. Which mean the app-id isn't appropriate.